### PR TITLE
Fix 134

### PR DESCRIPTION
**Pull request by joleroi**

### CODE REVIEW

File: regions/io/ds9/tests/test_ds9_language.py [hunk 1]
```diff
@@ -203,3 +203,10 @@ def test_ds9_color_override_global():
     assert regions[0].meta['color'] == 'green'
     assert regions[1].meta['color'] == 'orange'
     assert regions[2].meta['color'] == 'blue'
+
+def test_issue134_regression():
+    regstr = 'galactic; circle(+0:14:26.064,+0:00:45.206,30.400")'
+    parser = DS9Parser(regstr)
+    parser.run()
+    regions = parser.shapes
+    assert regions[0].to_region('radius').value == 30.4
```
Comment (joleroi) [inline regions/io/ds9/tests/test_ds9_language.py]: Shouldn't this be
```
regions[0].to_region().radius.value
```
?

Comment (keflavich) [inline regions/io/ds9/tests/test_ds9_language.py]: yes.  Might explain why it didn't pass...

File: regions/io/ds9/tests/test_ds9_language.py [hunk 2]
```diff
@@ -196,3 +203,10 @@ def test_ds9_color_override_global():
     assert regions[0].meta['color'] == 'green'
     assert regions[1].meta['color'] == 'orange'
     assert regions[2].meta['color'] == 'blue'
+
+def test_issue134_regression():
+    regstr = 'galactic; circle(+0:14:26.064,+0:00:45.206,30.400")'
```
Comment (keflavich) [inline regions/io/ds9/tests/test_ds9_language.py]: @joleroi this is a little redundant with your test, but it is a different case b/c it's `;`-separated, so I think it's reasonable to include.

Comment (joleroi) [inline regions/io/ds9/tests/test_ds9_language.py]: @keflavich you or me?
https://travis-ci.org/astropy/regions/jobs/233712956

**Comment by joleroi** [comment]
@keflavich you or me?
https://travis-ci.org/astropy/regions/jobs/233712956

**Comment by keflavich** [comment]
I don't see how any of our changes could have caused that issue.

**Comment by bsipocz** [comment]
@joleroi @keflavich - Neither of you, this is one of the sphinx 1.6.1 issues I've sent the e-mail out. There is a workaround now in ci-helpers, so the restarted build should pass (or fail with something related to this PR).

**Comment by joleroi** [comment]
Thanks @bsipocz , merging

**Pull request merged**